app/views/issues/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<% content_for :heading do %>
+  <h1><%= t ".title" %></h1>
+<% end %>
+
+<%= form_tag(issues_path, :method => :get) do %>
```

Comment:
I think this should be a POST form as currently it leads to very ugly URLs which doesn't seem very nice?

Also the form doesn't seem to preserve state so I can't tell what I'm currently filtering on and I have to start from scratch each time if I want to adjust my filter.